### PR TITLE
Update bunkerweb-install.sh script with the new NGINX (1.26.1) and BunkerWeb (1.5.8) versions

### PR DESCRIPTION
**Pull request by TheophileDiot**
## Description

Updated BunkerWeb to the latest version (1.5.8) which requires NGINX (1.26.1).
I also added the `apt-mark hold nginx bunkerweb` command back to avoid potential issues with automatic upgrades.

## Type of change

Please delete options that are not relevant.

- [x] I have performed a self-review of my code, adhering to established codebase patterns and conventions.

### CODE REVIEW

File: install/bunkerweb-install.sh [hunk 1]
```diff
@@ -23,19 +23,19 @@ $STD apt-get install -y lsb-release
 $STD apt-get install -y debian-archive-keyring
 msg_ok "Installed Dependencies"
 
-msg_info "Installing Nginx v1.20.0"
-wget -qO- https://nginx.org/keys/nginx_signing.key | gpg --dearmor >/usr/share/keyrings/nginx-archive-keyring.gpg
-echo "deb [signed-by=/usr/share/keyrings/nginx-archive-keyring.gpg] http://nginx.org/packages/debian `lsb_release -cs` nginx" >/etc/apt/sources.list.d/nginx.list
+msg_info "Installing Nginx v1.26.1"
+curl https://nginx.org/keys/nginx_signing.key | gpg --dearmor | $STD tee /usr/share/keyrings/nginx-archive-keyring.gpg >/dev/null
```
Comment (tteck) [inline install/bunkerweb-install.sh:27]: why would you change the commands?

Comment (TheophileDiot) [reply install/bunkerweb-install.sh:27]: I mirrored our official documentation: https://docs.bunkerweb.io/latest/integrations/#__tabbed_1_1

**Comment by tteck** [comment]
Look over the latest Bunkerweb commits to see if it meets your approval.

**Comment by tteck** [comment]
Despite the package being held, it still shows as upgradable, which is unusual.

To get around this, I had to use a pinning mechanism to prevent `apt` from considering the newer versions (testing).

```
cat <<EOF >/etc/apt/preferences.d/bunkerweb
Package: bunkerweb
Pin: version ${RELEASE}
Pin-Priority: 1001
EOF
```

**Comment by tteck** [comment]
As long as your future BunkerWeb releases continue to utilize the latest NginX releases, this configuration should work effectively.

**Comment by tteck** [comment]
Completed

**Comment by TheophileDiot** [comment]
@tteck I just checked your commit and everything looks fine except for the update part. 
We pin a specific NGINX version that can change from one BunkerWeb version to another (and it's not especially the latest). So it may create issues later :sweat_smile:

**Pull request closed without merging**